pkg/session_recording/recordings.jsx

```diff
@@ -238,17 +259,74 @@
                             navigateToItem={this.navigateToRecording.bind(this, r)}/>);
             }
             return (
-                <Listing.Listing title={_("Sessions")}
-                                 columnTitles={columnTitles}
-                                 emptyCaption={_("No recorded sessions")}
-                                 fullWidth={false}>
-                    {rows}
-                </Listing.Listing>
+                <div>
+                    <div className="content-header-extra">
+                        <table class="form-table-ct">
+                            <tr>
+                                <td className="top"><label className="control-label" for="dateSince">Date Since</label></td>
```

Comment:
Please wrap this line on 120 columns boundary to comply with [Cockpit coding guidelines](https://github.com/cockpit-project/cockpit/wiki/Cockpit-Coding-Guidelines).